aten/src/ATen/core/function_schema.cpp

```diff
@@ -485,7 +485,7 @@ bool FunctionSchema::isForwardCompatibleWith(
       return false;
     }
 
-    auto default_val = arguments().at(i).default_value().value();
+    auto const &default_val = arguments().at(i).default_value().value();
```

Comment:
?

Reply:
prevents unnecessary copy of the value out of std optional, I'm guessin.

Reply:
Exactly.